src/CI3Compatible/Database/CI_DB_query_builder.php

```diff
@@ -377,6 +403,8 @@ private function prepareSelectQuery(): void
         foreach ($this->order_by as $params) {
             $this->builder->orderBy(...$params);
         }
+
+        $this->builder->groupBy($this->group_by);
```

Comment:
I think it would be better to pass the saved parameters directly to `groupBy()` like other methods.

Reply:
I think I understand, you are suggesting `$this->group_by` should contain `[$by, $escape]` ?

Reply:
Yes!